lib/src/presentation/bloc/movie_from_endpoint_bloc.dart

```diff
@@ -0,0 +1,68 @@
+import 'dart:async';
+
+import '../../core/bloc/i_bloc.dart';
+import '../../core/parameter/parameter_object.dart';
+import '../../core/util/data_state.dart';
+import '../../core/util/debouncer.dart';
+import '../../domain/entity/movie.dart';
+import '../../domain/usecase/implementation/fetch_movies_from_endpoint_usecase.dart';
+
+class MoviesFromEndpointBloc implements IBloc {
+  static const titlePopular = 'Popular';
+  static const titleUpcoming = 'Upcoming';
+  static const titleTopRated = 'Top Rated';
+  static const titleNowPlaying = 'Now Playing';
+  static const titleNullEndpoint = 'NULL ENDPOINT';
```

Comment:
This scenario is possible to ocurrs?

Reply:
whoops, that was leftover from an old implementation. Fixed